File: C Programming/Function.c
#include<stdio.h>

   int sum(int a,int b,int c)
{
    return a+b+c;

}
    int sub(int a,int b,int c)
{
    return a-b-c;

}
int main()
{
    int result = sum(5,5,7);

    printf("The Sum is = %d\n",result);

    result = sub(19,6,7);
    printf("The  Sub is = %d\n",result);


}

File: JavaScrift/Function.js
function add(N1,N2){
    var result=N1+N2;
    return result;
  }

  var N=add(2,2);
  document.write(N +"</br>");

  var M=add(10,2);
  document.write(M);

  var a= parseFloat(prompt("Enter valur of a:"));
  var b= parseFloat(prompt("Enter valur of b:"));

(function sum(){

    var sum= a+b;
    document.write("Sum of your value="+ sum);
})();